sickbeard/tv.py

```diff
@@ -1448,38 +1448,47 @@ def downloadSubtitles(self, force=False):
                 languages.add(babelfish.Language.fromietf(language))
 
             if not languages:
-                logger.log(u"%s: No missing subtitles for S%02dE%02d" % (self.show.indexerid, self.season, self.episode), logger.DEBUG)
+                logger.log(u'%s: No missing subtitles for S%02dE%02d' % (self.show.indexerid, self.season, self.episode), logger.DEBUG)
                 return
 
             providers = sickbeard.subtitles.getEnabledServiceList()
-            videos = subliminal.scan_videos([self.location], subtitles=not force, embedded_subtitles=not sickbeard.EMBEDDED_SUBTITLES_ALL or not force)
-
-            if videos:
-                videos[0].tvdb_id = self.show.indexerid
-                videos[0].imdb_id = self.show.imdbid
-
-            # Lets create a second video object without hash, but with quality and release group
+            vname = self.location
+            create_link = False
             if self.release_name:
-                orig_fname = u'%s.%s' % (self.release_name, self.location.rsplit('.', 1)[1])
                 try:
-                    video = subliminal.video.Episode.fromname(orig_fname)
-                except ValueError:
-                    try:
-                        video = subliminal.video.Episode.fromname(self.release_name)
-                    except ValuError:
-                        pass
+                    dir, name = os.path.split(self.location)
+                    tmp = ek.ek(os.path.join, dir, self.release_name)
+                    if not len(tmp.splitext()[-1]):
+                        tmp += os.path.splitext(self.location)[-1]
+
+                    create_link = not ek.ek(os.path.exists, tmp)
+                    if create_link:
+                        ek.ek(helpers.link, self.location, tmp)
+                    vname = tmp
+                except:
+                    create_link = False
+                    vname = self.location
+                    pass
```

Comment:
Yes as it send an OSError: (38, 'Function not implemented').

Also you should use `except Exception:` instead of `except:` as you don't want to silence the KeyboardInterrupt(Ctrl+C to stop the program).